Edit badges in README (#289)
KinsonDigital/Velaptor: README.md

![GitHub Workflow Status](https://img.shields.io/github/workflow/status/kinsondigital/velaptor/%F0%9F%9A%80Preview%20Release?color=%23238636&label=Preview%20Release%20%F0%9F%9A%80&logo=github)
![Nuget (with prereleases)](https://img.shields.io/nuget/vpre/kinsondigital.velaptor?label=Latest%20Release&logo=nuget)
</div>

<div align="center">

![GitHub Workflow Status](https://img.shields.io/github/workflow/status/kinsondigital/velaptor/%E2%9C%94Unit%20Testing%20Status%20Check?color=%23238636&label=Unit%20Tests)
![](https://img.shields.io/codecov/c/github/KinsonDigital/Velaptor/master?label=Code%20Coverage&logo=CodeCov&style=flat)
![GitHub issues by-label](https://img.shields.io/github/issues/kinsondigital/velaptor/good%20first%20issue?color=%23238636&label=Good%20First%20Issues)
</div>

<div align="center">

![Discord](https://img.shields.io/discord/481597721199902720?color=%23575CCB&label=discord&logo=discord&logoColor=white)
![Twitter URL](https://img.shields.io/twitter/url?color=%235c5c5c&label=Follow%20%40KDCoder&logo=twitter&url=https%3A%2F%2Ftwitter.com%2FKDCoder)
</div>

<h2 style="font-weight:bold;border:0" align="center" >!! NOTICE !!</h2>

> This library is still under development and is not at v1.0.0 yet!!  All of the major features are there though so we encourage you to use it and report back any issues and improvements you may have.  That is what open source is all about!!

Reviewer: SS
```suggestion
This library is still under development and is not at v1.0.0 yet!!  However, all of the major features are available, so we encourage you to use it and report back any issues and improvements you may have.  
```

Reviewer: Did you want to keep "That is what open source is all about? "

<h2 style="font-weight:bold;border:0" align="center">📖 About Velaptor</h2>

> Velaptor is a 2D game development framework written in modern C# that strives to be simple and easy to use.  Game development can be difficult.  This is why using a framework that takes care of all of the lower level, more complicated things for creating a game can help!!  Things such as playing sound, loading graphics, managing content, multi-platform, and dealing with input are just some of the things that a game framework can provide.  This way you can just concentrate on developing your game or multi-media application.
> 

Reviewer: Reword
```suggestion
Velaptor is a 2D game development framework written in modern C# that strives to be simple and easy to use.  Game development can be difficult which is why Velaptor was developed.  It takes care of all of the lower level, more complicated things for creating a game like playing sound, loading graphics, managing content, multi-platform, and dealing with input, just to name a few.  This way you can concentrate on developing your game or multi-media application.

```

<h2 style="font-weight:bold;border:0" align="center">✨Features</h2>

> ### Cross-Platform

Reviewer: To set this feature apart from the text, you might want to bold it or make it bigger.
```suggestion
### Cross-Platform
```

> **Velaptor** has the ability to load texture atlas image and JSON data for texture atlas type rendering as well as single image content and sound content.  The content loading system is flexible and has an API that has the ability to be extended to load custom content for your media applications and games.

Reviewer: reword
```suggestion
**Velaptor** has the ability to load texture atlas images and JSON data for texture atlas type rendering as well as single image and sound content.  The content loading system is flexible and has an API with the ability to be extended to load custom content for your media applications and games.
```

> ### Content Caching

Reviewer: To set this feature apart from the text, you might want to bold it or make it bigger.
```suggestion
### Content Caching
```

> Loaded content such as images and sounds are cached for performance reasons.  If the same content attempting to be loaded from disk after it has already been loaded, that same content will be used from memory instead of being reloaded.  This is the same for sounds and texture atlas data.  Custom content loading will not come with caching and will have to be added manually.

Reviewer: reword
```suggestion
Loaded content such as images and sounds are cached for performance reasons.  If the same content is attempting to be loaded from a disk after it has already been loaded, that same content will be used from memory instead of being reloaded.  Sounds and texture atlas data have the same functionality.  Custom content loading will not come with caching and will have to be added manually.
```

Reviewer: You missed the "a" in front of "disk"

> ### Feature Demo Application

Reviewer: To set this feature apart from the text, you might want to bold it or make it bigger.
```suggestion
### Feature Demo Application
```

https://user-images.githubusercontent.com/85414302/150527337-6d872768-73dc-4603-82e8-37c691c78d4d.mp4

<h2 style="font-weight:bold;border:0" align="center">🔧Maintainers</h2>

We currently have the following maintainers:
- [Calvin Wilkinson](https://twitter.com/KDCoder) [<img src="https://about.twitter.com/etc/designs/about2-twitter/public/img/favicon.ico" alt="Follow Calvin Wilkinson on Twitter" width="16" />](https://twitter.com/KDCoder) (GitHub Organization / Owner)
- [Kristen Wilkinson](https://twitter.com/kswilky) [<img src="https://about.twitter.com/etc/designs/about2-twitter/public/img/favicon.ico" alt="Follow Calvin Wilkinson on Twitter" width="16" />](https://twitter.com/KDCoder) (GitHub Organization / Documentation Maintainer / Tester)

<h2 style="font-weight:bold;border:0" align="center">📄Documentation</h2>

> - Goto the [Table Of Contents](./Documentation/TableOfContents.md) for instructions on various things such as the branching, release process, environment setup and more.

Reviewer: Grammar
```suggestion
- Go to the [Table Of Contents](./Documentation/TableOfContents.md) for instructions on various topics such as: branching, release process, environment setup and more.
```

Reviewer: I like it!

<h2 style="font-weight:bold;border:0" align="center">🙏🏼Contributing</h2>

**Velaptor** encourages and uses [Early Pull Requests](https://medium.com/practical-blend/pull-request-first-f6bb667a9b6). Please don't wait until you're finished with your work before creating a PR.

> 

Reviewer: Make sure you put periods after all of the sentences. 
1. Fork the repository.
7.  Refer to the editorconfig files in the code base for rules.

> If you have any questions, contact a project maintainer.

Reviewer: ```suggestion
If you have any questions, please reach out to a project maintainer.
```

<h2 style="font-weight:bold;border:0" align="center">Practices</h2>

> - The code base is highly tested using unit testing while maintaining a high level of code coverage.  Manual testing is performed using the included testing application built specifically for manually testing the library.  When contributing, make sure to add or adjust the unit tests appropriately regarding your changes and perform manual testing.

Reviewer: Spelling
```suggestion
- The code base is highly tested by using unit tests while maintaining a high level of code coverage.  Manual testing is performed using the included testing application built specifically for manually testing the library.  When contributing, make sure to add or adjust the unit tests appropriately regarding your changes and perform manual testing.
```

Reviewer: My changes aren't reflected and I pulled down the most recent.

  - [Branching Diagram (GitHub Dark Mode)](./Documentation/Images/BranchingDiagram-DarkMode.png)
  - [Branching Diagram (GitHub Light Mode)](./Documentation/Images/BranchingDiagram-LightMode.png)

<h2 style="font-weight:bold;border:0" align="center">Further Resources</h2>

- The sample project named **VelaptorTesting** can be found in the [Testing Folder](https://github.com/KinsonDigital/Velaptor/tree/preview/master/Testing/VelaptorTesting)

>   - This is a sample project for the purpose to do manual testing of the library as well as to help understand how to use the library.

Reviewer: Reword
```suggestion
  - This is a sample project for the purpose of performing manual testing of the library as well as helping understand how to use it.
```
